data/human/far north jobs.txt

```diff
@@ -90,10 +90,12 @@ mission "Navy resupply [0]"
 	to offer
```

Comment:
```suggestion
	deadline
	to offer
```
I also just found out that there is no deadline on this mission where they should be.

~~That was my lowest quality PR yet. Oh well.~~